examples/comparison_pfd_vs_afd.py

```diff
@@ -0,0 +1,32 @@
+import desbordante
+
+TABLE = 'examples/datasets/glitchy_sensor.csv'
+ERROR = 0.18
+ERROR_MEASURE = 'per_value' # per_tuple or per_value
+
+
+def make_set(fds):
+    return set(map(str, fds))
+
+
+def get_afds():
+    algo = desbordante.afd.algorithms.Tane()
+    algo.load_data(table=(TABLE, ',', True))
+    algo.execute(error=ERROR)
+    return algo.get_fds()
+
+
+def get_pfds():
+    algo = desbordante.pfd.algorithms.PFDTane()
+    algo.load_data(table=(TABLE, ',', True))
+    algo.set_option('error_measure', ERROR_MEASURE)
+    algo.execute(error=ERROR)
+    return algo.get_fds()
+
+
+pfds = make_set(get_pfds())
+afds = make_set(get_afds())
+
+print("pFDs \ AFDs =", pfds - afds)
```

Comment:
why you write here "pFDs", not "PFDs"? I believe that it is better to use "PFDs", similar to `AFDs`

Reply:
it's from the paper http://i.stanford.edu/~anishds/publications/webdb09/webdb09.pdf